flask_admin/contrib/sqla/view.py

```diff
@@ -828,11 +830,11 @@ def _apply_search(self, query, count_query, joins, count_joins, search):
                                                                                    inner_join=False)
 
                 column = field if alias is None else getattr(alias, field.key)
-                filter_stmt.append(column.ilike(stmt))
+                filter_stmt.append(cast(column, Unicode).ilike(stmt))
```

Comment:
My first reaction to this was "that looks bad for performance". But, I checked and MySQL isn't using an index for these searches anyway, because we're using `LIKE lower("%value%")`. I'm not noticing a speed difference before/after the change when doing a search on a datetime column with 50k rows.

I confirmed it prevents the exception when searching INET columns (and other columns?), so it seems like a good change.